index.html

```diff
@@ -168,7 +168,29 @@ <h4>LET'S CONNECT</h4>
         <li><img src="./images/Frame12.svg" alt=""></li>
       </ul>
     </div>
-  </div> 
-</section>   
+  </div>
+   <div class="contact-main">
+    <div class="contact-me">
+      <div class="text-contact">
+        <img id="contact-img" src="./images/PrimaryText-1.svg" alt="Contact-me">
+        <br>
+        <img src="./images/PrimaryText-21.svg" alt="text">
+      </div>
+      <div class="form">
+        <form action="https://formspree.io/f/mknyrjvn" method="post">
+        <ul>
+          <li><input type="text" maxlength="30" placeholder="Name" required></li>
+          <li><input type="email" placeholder="Enter E-mail" required></li>
```

Comment:
- [ ] Great job implementing the form, however, it's one of the project [requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/3_contact_form.md#project-requirements) to place the form in a footer it will be more semantic.

![image](https://user-images.githubusercontent.com/59416136/154128237-dcee89fa-3ccc-4f66-bd21-33c20d48d8c0.png)